src/main/kotlin/app/revanced/patches/youtube/layout/hidesubtitlespopup/patch/HideSubtitlesPopupPatch.kt

```diff
@@ -0,0 +1,30 @@
+package app.revanced.patches.youtube.layout.hidesubtitlespopup.patch
+
+import app.revanced.extensions.toErrorResult
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.replaceInstructions
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.youtube.layout.hidesubtitlespopup.annotation.HideSubtitlesPopupPatchCompatibility
+import app.revanced.patches.youtube.layout.hidesubtitlespopup.fingerprint.HideSubtitlesPopupFingerprint
+
+@Patch
+@Name("hide-subtitles-popup")
```

Comment:
Please reflect the changes to the class and package name as well.
```suggestion
@Name("hide-subtitles-toggle-toast")
```